src/create_disk.sh

```diff
@@ -544,16 +544,58 @@ create_dmverity() {
     mount -o ro "/dev/mapper/${partlabel}" "${mountpoint}"
 }
 
+# Save genprotimg input for later and don't run zipl here
+rdcore_replacement() {
+    local se_kargs_append=("ignition.firstboot")
+    while [ $# -gt 0 ]; do
+        se_kargs_append+=("$1")
+        shift
+    done
+
+    local se_tmp_disk se_tmp_mount blsfile
+    local se_script_dir="/usr/lib/coreos-assembler/secex-genprotimgvm-scripts"
+    se_tmp_disk=$(realpath /dev/disk/by-id/virtio-genprotimg)
+    se_tmp_mount=$(mktemp -d /tmp/genprotimg-XXXXXX)
+    local se_tmp_boot="${se_tmp_mount}/genprotimg"
+    mount "${se_tmp_disk}" "${se_tmp_mount}"
+    mkdir "${se_tmp_boot}"
+
+    local se_initrd="${se_tmp_boot}/initrd.img"
+    local se_kernel="${se_tmp_boot}/vmlinuz"
+    local se_parmfile="${se_tmp_boot}/parmfile"
+
+    blsfile=$(find "${rootfs}"/boot/loader/entries/*.conf)
+    echo "$(grep options "${blsfile}" | cut -d' ' -f2-)" "${se_kargs_append[@]}" > "${se_parmfile}"
+    kernel="${rootfs}/boot/$(grep linux "${blsfile}" | cut -d' ' -f2)"
+    initrd="${rootfs}/boot/$(grep initrd "${blsfile}" | cut -d' ' -f2)"
+    cp "${kernel}" "${se_kernel}"
+    cp "${initrd}" "${se_initrd}"
+
+    # genprotimg and zipl will be done outside this script
+    # copy scripts for that step to the tmp disk
+    cp "${se_script_dir}/genprotimg-script.sh" "${se_script_dir}/post-script.sh" "${se_tmp_mount}/"
+
+    umount "${se_tmp_mount}"
+    rm -rf "${se_tmp_mount}"
```

Comment:
Nit: since we expect the directory to be empty, `rmdir` would be cleaner, since it will fail if our expectation is wrong.